### DISCUSSION TITLE
Expiry dates for menu links

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-future#368] [Feature Requests]
> I’ve set a page on a website to expire to PRIVATE on a specific date, however, as I understand it, any menu items pointing to that page will remain on the website navigation menus.
> 
> Including a checkbox settings like ‘Remove from all navigation menus on expire’ would be very useful indeed.
> 

https://wordpress.org/support/topic/suggestion-remove-from-all-navigation-menus-on-expire/

**Comment by ZoltanGalCMDesign** [comment]
+1
